src/locales/en.json

```diff
@@ -1103,7 +1106,6 @@
 "sshKeyPairs": "SSH keypairs",
 "wednesday": "Wednesday",
 "noselect": "No thanks",
-"group": "Group",
```

Comment:
`group` is used in VM wizard, check and unremove it @shwstppr

Reply:
already present above in the file
https://github.com/apache/cloudstack-primate/blob/fd882b35a9a547f8a104d5ddeef34b9058ccb9a8/src/locales/en.json#L234
this was duplicate @rhtyd